app/views/core/RootView.coffee

```diff
@@ -188,7 +188,6 @@ module.exports = class RootView extends CocoView
     @saveLanguage(newLang)
     locale.load(me.get('preferredLanguage', true)).then =>
       @onLanguageLoaded()
-      window.tracker.promptForCookieConsent()
```

Comment:
Is the prompt for cookie consent handled elsewhere on a language change?

Reply:
Hmm no it's not.  I'd initially thought that this was a hack to work around the proper locale being loaded but looking again it could be how we trigger the cookie consent screen to rerender when language changes.  I'm going to update the cookie consent tracker to support this.

Reply:
Done